library/src/main/java/com/chuckerteam/chucker/internal/data/entity/HttpTransactionTuple.kt

```diff
@@ -21,7 +21,8 @@ internal class HttpTransactionTuple(
     @ColumnInfo(name = "responseCode") var responseCode: Int?,
     @ColumnInfo(name = "requestContentLength") var requestContentLength: Long?,
     @ColumnInfo(name = "responseContentLength") var responseContentLength: Long?,
-    @ColumnInfo(name = "error") var error: String?
+    @ColumnInfo(name = "error") var error: String?,
+    @ColumnInfo(name = "isSlowApiCall") var isSlowApiCall: Boolean = false
```

Comment:
Same as above

Reply:
Since we are not storing the `normalAPIDuration` i believe we need this boolean field.
Or are you suggesting we save `normalAPIDuration` instead of the boolean?